manifests/params.pp

```diff
@@ -18,16 +17,44 @@
 
   case $::osfamily {
     'Debian': {
-      $libdir           = '/usr/share/jenkins'
-      $package_provider = 'dpkg'
+      $jenkins_user       = 'jenkins'
+      $jenkins_group      = 'jenkins'
+      $repo               = true
+      $homedir            = '/var/lib/jenkins'
+      $jenkinsdir         = $homedir
+      $default_plugin_dir = "${jenkinsdir}/plugins"
+      $libdir             = '/usr/share/jenkins'
+      $package_provider   = 'dpkg'
     }
     'RedHat': {
-      $libdir           = '/usr/lib/jenkins'
-      $package_provider = 'rpm'
+      $jenkins_user       = 'jenkins'
+      $jenkins_group      = 'jenkins'
+      $repo               = true
+      $homedir            = '/var/lib/jenkins'
+      $jenkinsdir         = $homedir
+      $default_plugin_dir = "${jenkinsdir}/plugins"
+      $libdir             = '/usr/lib/jenkins'
+      $package_provider   = 'rpm'
+    }
+    'OpenBSD': {
+      $jenkins_user       = '_jenkins'
+      $jenkins_group      = '_jenkins'
+      $repo               = false
+      $homedir            = '/var/jenkins'
+      $jenkinsdir         = "${homedir}/.jenkins"
```

Comment:
I'm not the jenkins package maintainer. But as far as I can see, it's the way it is since ever on OpenBSD.
What is done is just using downloading the war file, deployed to /usr/local/share/jenkins. No JENKINS_HOME or the like ENV variable is set. As far as I can see, without such variable, the war file defaults to the ~/.jenkins directory of the jenkins user?